packages/iris-grid/src/IrisGrid.jsx

```diff
@@ -3209,6 +3219,8 @@ IrisGrid.propTypes = {
 
   // Theme override for IrisGridTheme
   theme: PropTypes.shape({}),
+
+  isSearchBarToggleVisible: PropTypes.bool.isRequired,
```

Comment:
Don't make this `.isRequired`, just add a default to `true`

Reply:
May as well name it `canToggleSearch` to conform with `canCopy`/`canDownloadCsv`